sqlitedb/queries.md

```diff
@@ -0,0 +1,37 @@
+# The Sqlite db was generated from testnet data as below:
+
+- Login into the sqlite3 CLI shell by:
+```Shell
+    cd /path/to/testnet3/sqlite/db
+
+    sqlite3
+```
+
+- While in the sqlite3 shell connect to the testnet db.
+```Shell
+   .open dcrdata.sqlt.db
+
+    create table test_dcrdata_block_summary as select * from dcrdata_block_summary where height > 131800 LIMIT 200;
+    create table test_dcrdata_stakeinfo_extended as select * from dcrdata_stakeinfo_extended where height > 131800 LIMIT 200;
+```
+
+- While in the sqlite shell create a dump file with the newly created tables's data.
```

Comment:
While in the sqlite shell,[**<-comma**] create **dump files** with the newly created **tables'** data.